Guide.docc/IncrementalAdoption.md

```diff
@@ -149,6 +149,33 @@ class UIStyler {
 Combining static and dynamic isolation can be a powerful tool to keep the
 scope of changes gradual.
 
+### Manual MainActor Isolation
+
+The `assumeIsolated` method is synchronous and exists to express isolation
+that would otherwise be invisible to the compiler.
+The `MainActor` type also has a method you can use to manually switch
+isolation in an asynchronous context.
+
+```swift
+// type that should be MainActor, but has not been updated yet
+class PersonalTransportation {
+}
+
+await MainActor.run {
+    // isolated to the MainActor here
+    let transport = PersonalTransportation()
+    
+    // ...
+}
+```
+
```

Comment:
Maybe also do a note here that "the preferable solution would be to annotate PersonalTransportation with@MainActor?

Reply:
yes agreed!